### ISSUE TITLE
If backtrack is >800, start ignoring the last ticks.

### ISSUE THREAD

**Issue by SylveonBottle** [nullworks/cathook#733]
Setting backtrack to 1000 creates a whole row of backtrack ticks that can't actually be used due to how servers work.

When you're at 1000 backtracking, only the newest tick will be taken by the server. Any more than that is actually beyond what's acceptable. 

If you wanna test for yourself, just set backtracking to 1000 and watch how every single one of your shots start missing - including backstabbing, sniping, etc. Then lower it to `(800 - yourActualPing)`and notice how all shots start hitting.

I could be BS'ing this entire thing out of my ass, but from experience having backtrcking set to 840 for me results in the entire back half of the green line (with `backtrack.set`) results in no hits, while the front half do hit. This is with my ping at 60 - which results in an "actual backtracking" back-ticks of 900-1100, whereas all hits beyond the 1000 threshold aren't registered.

Basically: If the user sets their backtracking between 800-900, let's throw away the last half. If it's set to 1000, let's ignore all but the latest tick.

**Comment by BenCat07** [comment]
Actually i even calculate ping in backtrack i'm pretty sure so we use valid ticks? TODO: check if We get the current ping wrong

**Comment by SylveonBottle** [comment]
Was this "fixed" with https://github.com/nullworks/cathook/commit/fe1771cdc6e2ed81f56d3d5381938e7d72e99508 ?

**Comment by BenCat07** [comment]
Yes
